src/commands/generate/types.ts

```diff
@@ -395,3 +376,9 @@ export function getVariableType(variable: Variable) {
     }
     return variable.type.toLocaleLowerCase()
 }
+
+function isVariableDeprecated(variable: Variable, features: Feature[]) {
+    if (!variable._feature || variable.persistent) return { deprecated: false}
+    const feature = features.find(f => f._id === variable._feature)
+    return {deprecated: feature && feature.status !== 'active', feature}
```

Comment:
wouldn't this catch `in progress` features?

Reply:
there are only three statuses, active, complete and archived. It's also only fetching completed and archived features to begin with

Reply:
kk, was just looking at the ones we list on the dashboard: 
<img width="297" alt="image" src="https://github.com/user-attachments/assets/4d872829-4f8b-462e-bb53-19bfc67492b1">